src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -2069,7 +2072,8 @@ public WorkspacePreferences getWorkspacePreferences() {
                 getBoolean(OPEN_LAST_EDITED),
                 getBoolean(SHOW_ADVANCED_HINTS),
                 getBoolean(WARN_ABOUT_DUPLICATES_IN_INSPECTION),
-                getBoolean(CONFIRM_DELETE));
+                getBoolean(CONFIRM_DELETE),
+                new HashSet<>(getStringList(SELECTED_SLR_FETCHERS)));
```

Comment:
why not using a list of fetchers? has the advantage that their order is the same